tools/c7n_gcp/tests/data/flights/node-template-delete/get-compute-v1-projects-cloud-custodian-aggregated-nodeTemplates_1.json

```diff
@@ -0,0 +1,276 @@
+{
+  "body": {
+    "items": {
+      "regions/us-west2": {
```

Comment:
i've added a separate pr #4615 with a script to that can be used to strip all the junk empty responses in the recorded data here.

Reply:
done